Homework/src/io/codelex/arithmetic/practice/ExerciseEight.java

```diff
@@ -0,0 +1,18 @@
+package io.codelex.arithmetic.practice;
+
+public class ExerciseEight {
+    public static void main(String[] args) {
+        int hoursWorked = 57;
+        double payRate=9.5;
+        calculatePay(hoursWorked, payRate);
+    }
+    static void calculatePay(int num, double rate){
+        if(rate<8) System.out.println("Rate is lower than the State minimum. Pay more!");
```

Comment:
Ok pie trešā nenoformatētā faila es jau Tev ieteiktu ielikt intellij pie save actions, lai viņš to noformatē :D Vari google sameklēt kā to izdarīt.
Kā arī - pie IF jālieto figūriekavas, turpmāk to nekomentēšu - izlabo vienkārši visur.